packages/projection/src/operators/Mappers/withHandles.ts

```diff
@@ -53,13 +68,19 @@ const getBurnedHandles = (mint: Cardano.TokenMap | undefined, policyIds: FilterB
     if (quantity < 0n) {
       const policyId = Cardano.AssetId.getPolicyId(assetId);
       if (!policyIds.includes(policyId)) continue;
-      const handle = handleFromAssetId(assetId);
-      handles[handle] = {
-        assetId,
-        handle,
-        latestOwnerAddress: null,
-        policyId
-      };
+      try {
+        const handle = handleFromAssetId(assetId);
```

Comment:
Seems we have some code duplication here. Please hoist to DRY the code

Reply:
Fixed on 225763b